tests/unit/select_test.coffee

```diff
@@ -344,3 +344,14 @@ test 'optionValuePath with nested valuePath', ->
 
   equal obj2, select.get('selection'),
     'The right selection is retrieved'
+
+test 'shouldEnsureVisible controls whether to ensure visibility', ->
+  expect 0
+
+  select = @subject
+    content: ['foo']
+    shouldEnsureVisible: no
+    ensureVisible: ->
+      ok no, 'ensureVisible is not called if shouldEnsureVisible is false'
```

Comment:
should we use sinon.spy instead of this, and check if the spy is not called? Having `expect 0` is not natural